lib/model.js

```diff
@@ -958,6 +961,36 @@ class Model extends Entity {
     }
 }
 
+/**
+ * Parse the id and according to the keyType config in the Schema ("name"|"id"|<undefined>)
+ * it will convert an '123'(string) id to 123 (int).
+ * @param {*} id -- id passed in constructor
+ */
+const getParseId = (schema, gstore) => id => {
```

Comment:
I'd prefer to maintain `parseId()` for the function name.

Also, instead of attaching it to the `ModelInstance` above and declare it here, it might be better to directly declare it as a static method (right below `static getEntitiesRefsFromSchema()`. What do you think?

Reply:
Yep! The reason I did it that way was when parseId was used in map, it didn't get `this` passed through, but I just realized you can hand map a `this` to use. I just changed my local copy, gonna squash the commits and update the PR.